reana-ui/src/actions.js

```diff
@@ -57,11 +57,13 @@ const USER_SIGNUP_URL = `${api}/api/register`;
 const USER_SIGNIN_URL = `${api}/api/login`;
 const USER_SIGNOUT_URL = `${api}/api/logout`;
 const USER_REQUEST_TOKEN_URL = `${api}/api/token`;
-const WORKFLOWS_URL = ({ page = 1, size }) => {
-  let url = `${api}/api/workflows`;
-  if (size) {
-    url += `?page=${page}&size=${size}`;
-  }
+const WORKFLOWS_URL = ({ page = 1, size, search, status, sortDir }) => {
+  //TODO: use query-string to format query params
```

Comment:
Maybe we can include this dependency already? It shouldn't be a lot of work, should it?
If it's too much we can ticketize and leave it for later.

Reply:
Yeah, I think we can do it now. I left it as a todo item just to make a leaner PR. I will implement it asap

Reply:
If it's a separate commit it should be fine.